pykickstart/commands/url.py

```diff
@@ -237,6 +236,9 @@ def __str__(self):
 
     def _getParser(self):
         op = F18_Url._getParser(self)
+        op.add_argument("--url", version=F18, help="""
```

Comment:
I guess that there should be the version F27.

Reply:
oh my, sorry ...
Fixed.